controllers/applicationsnapshotenvironmentbinding_controller.go

```diff
@@ -49,10 +49,10 @@ import (
 // SnapshotEnvironmentBindingReconciler reconciles a SnapshotEnvironmentBinding object
 type SnapshotEnvironmentBindingReconciler struct {
 	client.Client
-	Scheme   *runtime.Scheme
-	Log      logr.Logger
-	AppFS    afero.Afero
-	Executor gitopsgen.Executor
+	Scheme *runtime.Scheme
+	Log    logr.Logger
+	AppFS  afero.Afero
+	// Executor gitopsgen.Executor
```

Comment:
Why's the executor being removed from all of the GitOps functions?

Reply:
This was from the main branch, rebased and it should be gone